migrations/versions/67ac3b7d5c2f_remove_column_faculty_reviewed_from_.py

```diff
@@ -0,0 +1,25 @@
+"""remove column faculty_reviewed from Nicety
+
+Revision ID: 67ac3b7d5c2f
+Revises: e88e0781857f
+Create Date: 2021-06-25 13:00:46.937589
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '67ac3b7d5c2f'
+down_revision = 'e88e0781857f'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column('nicety', 'faculty_reviewed')
+    # ### end Alembic commands ###
+
+
+def downgrade():
```

Comment:
I wasn't able to run the migration, because the definition for `downgrade` is incomplete. It either needs to have a `pass` as its body, or be omitted completely. I'd prefer the latter, and Alembic seems to be okay with a missing `downgrade` method, so I suggest that:
```suggestion
```